eksctl/2i2c-aws-us.jsonnet

```diff
@@ -125,7 +156,28 @@ local daskNodes = [
                 "hub.jupyter.org/dedicated": "user:NoSchedule"
             },
         } + n for n in notebookNodes
-    ] + ( if daskNodes != null then
+    ] + ( if dedicatedNotebookNodes != null then
+        [
+        ng + {
```

Comment:
This can be removed by rolling the dedicated nodes directly into the `notebookNodes` definition.